packages/dashboards/examples/src/components/docs/tutorials/customization/customization.module.ts

```diff
@@ -1,5 +1,6 @@
 import { NgModule } from "@angular/core";
 import { RouterModule, Routes } from "@angular/router";
+import { ConfiguratorHeadingService } from "../../../../../../src/public-api";
```

Comment:
Please change to `import { ConfiguratorHeadingService } from "@nova-ui/dashboards";`

Reply:
done